warehouse/organizations/models.py

```diff
@@ -538,6 +540,10 @@ class OrganizationApplication(OrganizationMixin, HasObservations, db.Model):
     __tablename__ = "organization_applications"
     __repr__ = make_repr("name")
 
+    @declared_attr
```

Comment:
did a giant double take wondering how https://github.com/pypi/warehouse/blob/193853502a523ccc1d2bfa5ecce0a1893b53e998/warehouse/admin/views/organizations.py#L362-L371 had been working, and realized it was via inheritance.